src/Qowaiv.Validation.TestTools/FluentAssertions/ResultAssertions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Qowaiv.Validation.Abstractions;
+
+namespace FluentAssertions.Qowaiv.Validation
+{
+    /// <summary>Contains a number of methods to assert the (in)validness of a <see cref="Result"/>.</summary>
+    public sealed class ResultAssertions : ResultAssertionsBase<Result>
+    {
+        /// <summary>Creates a new instance of the <see cref="ResultActExtensions"/> class.</summary>
```

Comment:
Is this comment correct?

Reply:
I think so, do you have a better/other idea?

Reply:
I was mostly wondering, here and in the other comments, if the class listed is actually the correct one (probably should have been clearer on this one 😅).

Reply:
I think so. Maybe, the naming could be better, and I'm open to suggestions, but the paths disallow `.WithoutMessages()` for invalid, and `.Value` is only accessible for `.Should().BeValid()` on `Result<T>`.

Reply:
I'm not sure if we are talking about the same thing here. The comment mentions creating an instance of `ResultActExtensions`, but the class here is `ResultAssertions`. But could be I'm missing something as it's a bit harder to trace stuff in the online diff view.

Reply:
Ill check if all of them match. :) I thought you're referring to meaning of the text, not the type mismatches.